src/vtkF3DAlembicImporter.h

```diff
@@ -0,0 +1,126 @@
+/**
+ * @class   vtkF3DAlembicImporter
+ * @brief   Importer using Alembic library
+ *
+ * This importer is based on Alembic 1.7
+ * @sa https://github.com/alembic/alembic/blob/master/README.txt
```

Comment:
Improve doc a bit if possible, especially regarding what is supported or not.

Reply:
Improved.